Kernel/ParallelTransport.m

```diff
@@ -277,6 +330,7 @@
 
 
 KerrParallelTransportFrameFunction[a_, p_, e_, x_, assoc_][\[Lambda]_/;StringQ[\[Lambda]] == False] := assoc["ParallelTransportedFrame"][\[Lambda]]
+KerrParallelTransportFrameFunction[a_, p_, e_, x_, assoc_][\[Lambda]__] := assoc["ParallelTransportedFrame"][\[Lambda]]
 KerrParallelTransportFrameFunction[a_, p_, e_, x_, assoc_][y_?StringQ] := assoc[y]
```

Comment:
We should add Keys support to KerrParallelTransportFrameFunction, just as we have for KerrGeoOrbitFunction

Reply:
Done